bokehjs/test/test_common/test_selection_manager.coffee

```diff
@@ -77,9 +77,6 @@ describe "SelectionManager", ->
 
   sm = new SelectionManager()
 
-  it "should have a reference to an empty selection", ->
-    expect(sm.empty).to.deep.equal empty_selection
```

Comment:
> We could also move this comment to a more appropriate issue

@birdsarah, what you wrote is mostly correct. I will comment and expand in a more appropriate place.